.gitignore

```diff
@@ -53,3 +53,5 @@ yarn-debug.log*
 
 /app/assets/builds/*
 !/app/assets/builds/.keep
+
+.idea
```

Comment:
This is unique to your specific environment so should be removed.  See the comment at the top for setting a global ignore for this.